api/src/main/java/com/netflix/iceberg/expressions/Expressions.java

```diff
@@ -117,6 +117,10 @@ public static Expression not(Expression child) {
     return new UnboundPredicate<>(op, ref(name));
   }
 
+  public static <T> UnboundPredicate<T> alwaysTrue(String name) {
```

Comment:
Why was this added? A `true` predicate has no field refence.

Reply:
I've added this method in `com.netflix.iceberg.expressions.Expressions` because `com.netflix.iceberg.expressions.UnboundPredicate` constructors are package-private so it's visible only within its own package and I needed to override method from class in different package `com.netflix.iceberg.transforms.UnknownTransform#project`. Does this answer your question? 
Since this method is obviously taking a redundant argument would you consider this version instead? 
```return new UnboundPredicate<>(Operation.TRUE, null);```